gallery/src/data-visualization/ReactVisBarGraphPage.tsx

```diff
@@ -0,0 +1,141 @@
+        <header className="nx-tile-subsection__header">
+          <h3 className="nx-h3">Basic API Reference</h3>
+        </header>
+        <NxTable>
+          <NxTableHead>
+            <NxTableRow>
+              <NxTableCell>Name</NxTableCell>
+              <NxTableCell>Type</NxTableCell>
+              <NxTableCell>Required</NxTableCell>
+              <NxTableCell>Description</NxTableCell>
+            </NxTableRow>
+          </NxTableHead>
+          <NxTableBody>
+            <NxTableRow>
+              <NxTableCell>barWidth</NxTableCell>
+              <NxTableCell>number/string/object</NxTableCell>
+              <NxTableCell>No</NxTableCell>
+              <NxTableCell>
+                The percentage for which each bar fills the designated bucket. 1.0 means that the bar fills the whole
+                bucket (no padding between bars), while a smaller percentage means more whitespace between the bars.
+              </NxTableCell>
+            </NxTableRow>
+            <NxTableRow>
+              <NxTableCell>color</NxTableCell>
+              <NxTableCell>string</NxTableCell>
+              <NxTableCell>No</NxTableCell>
+              <NxTableCell>
+                Assigns the specified color to the series.
+              </NxTableCell>
+            </NxTableRow>
+            <NxTableRow>
+              <NxTableCell>stroke</NxTableCell>
+              <NxTableCell>string</NxTableCell>
+              <NxTableCell>no</NxTableCell>
+              <NxTableCell>
+                In LineSeries, stroke is used instead of color to specify the color of the lines.
+              </NxTableCell>
+            </NxTableRow>
+            <NxTableRow>
+              <NxTableCell>tickFormat</NxTableCell>
+              <NxTableCell>function</NxTableCell>
+              <NxTableCell>No</NxTableCell>
+              <NxTableCell>
+                Format function for the tick label. Typically used to transform ticks to show addtional information.
+                For example, <code className="nx-code">{'tickFormat={val => `$${val}`}'}</code> adds a $ before
+                the tick label.
+              </NxTableCell>
+            </NxTableRow>
+            <NxTableRow>
+              <NxTableCell>title</NxTableCell>
+              <NxTableCell>string</NxTableCell>
+              <NxTableCell>No</NxTableCell>
+              <NxTableCell>
+                Shows the title for the axis.
+              </NxTableCell>
+            </NxTableRow>
+          </NxTableBody>
+        </NxTable>
+      </section>
+    </GalleryDescriptionTile>
+
+    <GalleryExampleTile title="Bar Graph Example"
+                        id="react-vis-bar-graph-example"
+                        liveExample={ReactVisBarGraphExample}
+                        codeExamples={reactVisBarGraphExampleCode}>
+      The following example demonstrates how to generate a simple vertical bar graph using
+      {' '} <code className="nx-code">{'<VerticalBarSeries />'}</code> <br />
```

Comment:
Same question as before about the `<br/>`

Reply:
> Same question as before about the `<br/>`

Has been removed.
https://github.com/sonatype/sonatype-react-shared-components/pull/442/files#diff-01f9e2b8feee08c87c5551bc7fcfd37908269fed4267f1a52de73e7c54d8ceb0R125-R132